src/lib/components/Swap/Input.tsx

```diff
@@ -1,17 +1,24 @@
 import { Trans } from '@lingui/macro'
 import { useUSDCValue } from 'hooks/useUSDCPrice'
+import { useAtomValue } from 'jotai/utils'
+import { loadingOpacityCss } from 'lib/css/loading'
 import { useSwapAmount, useSwapCurrency, useSwapInfo } from 'lib/hooks/swap'
 import { usePrefetchCurrencyColor } from 'lib/hooks/useCurrencyColor'
-import { Field } from 'lib/state/swap'
+import { Field, independentFieldAtom } from 'lib/state/swap'
 import styled, { ThemedText } from 'lib/theme'
 import { useMemo } from 'react'
+import { TradeState } from 'state/routing/types'
 import { formatCurrencyAmount } from 'utils/formatCurrencyAmount'
 
 import Column from '../Column'
 import Row from '../Row'
 import TokenImg from '../TokenImg'
 import TokenInput from './TokenInput'
 
+const LoadingSpan = styled.span<{ $loading: boolean }>`
+  ${loadingOpacityCss};
+`
+
```

Comment:
how does the $ prefix work here? and is the prop being used anywhere?

Reply:
not sure how props get passed to css interpolations, but assumed they were not

Reply:
we just cant use `loading` as its a reserved prop name, will give us TS errors

Reply:
cool that this functions. TIL.

Reply:
Leave a comment that `loading` is reserved, and that's why you use `$loading`. Otherwise someone will waste half an hour trying to "fix" it in a month.